docs/interactions/Receiving_and_Responding.md

```diff
@@ -311,6 +311,7 @@ We highly recommend checking out our [Community Resources](#DOCS_TOPICS_COMMUNIT
 ## Create Interaction Response % POST /interactions/{interaction.id#DOCS_INTERACTION_RECEIVING_AND_RESPONDING/interaction}/{interaction.token#DOCS_INTERACTION_RECEIVING_AND_RESPONDING/interaction-object}/callback
 
 Create a response to an Interaction from the gateway. Takes an [interaction response](#DOCS_INTERACTION_RECEIVING_AND_RESPONDING/interaction-response-object).
+This endpoint also supports file attachments similar to the webhook endpoints. Refer to [Uploading Files](#DOCS_RESOURCES_CHANNEL/create-message-uploading-files) for details on uploading files and `multipart/form-data` requests.
```

Comment:
Is this a case of you can include file attachments with the initial response for interactions received over the gateway but not for interactions received over REST?

Reply:
Since the file uploads haven't been documented for that response at all, I excluded it intentionally. Theoretically, it would work identically but with a multipart body response.